### DISCUSSION TITLE
How to train more than one model simultaneously?

### DISCUSSION THREAD

**Discussion by Xhao12138** [lululxvi/deepxde#1617] [Q&A]
How to train more than one model simultaneously?I mean,can I train two or more models at the same time?

**Comment by vl-dud** [comment]
If you are talking about parallelization, then yes, what is the actual problem? But if you mean defining models in one piece of code, then look at [https://github.com/lululxvi/deepxde/pull/1559](https://github.com/lululxvi/deepxde/pull/1559)

**Reply by Xhao12138** [reply]
Thank you for your reply!I mean defining models in one piece of code.And I am interested in your 'parallelization',what does it mean?

**Reply by vl-dud** [reply]
[https://deepxde.readthedocs.io/en/latest/user/parallel.html](https://deepxde.readthedocs.io/en/latest/user/parallel.html)
